chef/cookbooks/provisioner/templates/default/unattended.xml.erb

```diff
@@ -88,68 +88,72 @@
                     <Order>1</Order>
                 </SynchronousCommand>
                 <SynchronousCommand wcm:action="add">
+                    <CommandLine>netsh advfirewall firewall set rule name="File and Printer Sharing (SMB-In)" new enable=yes</CommandLine>
+		    <Order>2</Order>
```

Comment:
Why not add it at the end (to avoid changing the order of everything else)? Does it matter? (although I won't object to the patch just for this, just curious)